taipy/gui/gui.py

```diff
@@ -989,6 +989,8 @@ def __upload_files(self):
         context = request.form.get("context", None)
         upload_data = request.form.get("upload_data", None)
         multiple = "multiple" in request.form and request.form["multiple"] == "True"
+
```

Comment:
can you write some python test to ensure that __upload_files works the same before and after your changes ?

Reply:
Sure, I'll do that.

Reply:
I wrote a python test that covers the new functionality (uploading nested structures), but the upload of files (including multiple, missing files and various other options) is already tested by other tests in `tests/gui/server/http/test_file_upload.py`, which still pass. I can not think of other useful test cases, that I can further derive.
Maybe as a further explanation, why that is:
```
```